site/content/en/docs/Advanced/high-availability-agones.md

```diff
@@ -13,9 +13,9 @@ publishDate: 2023-02-28
 
 When `SplitControllerAndExtensions` is enabled, the `agones-controller` responsibility is split up into `agones-controller`, which enacts the Agones control loop, and `agones-extensions`, which acts as a service endpoint for webhooks and the allocation extension API. Splitting these responsibilities allows the `agones-extensions` pod to be **horizontally scaled**, making the Agones control plane **highly available** and more **resiliant to disruption**.
 
-{{< alert title="GKE Autopilot Clusters" color="warning" >}}
+`SplitControllerAndExtensions` also permits multiple `agones-controller` pods and leader eleciton usage. Having multiple `agones-controller` minimizes downtime of the service from voluntary pod disruptions such as pod deletes, deployment updates, and pod restarts.
```

Comment:
Suggestion:

```
`SplitControllerAndExtensions` enables multiple `agones-controller` pods, with a primary controller selected via leader election. Having multiple `agones-controller` minimizes downtime of the service from pod disruptions such as deployment updates, autoscaler evictions, and crashes.
```

(This actually helps the case of involuntary disruptions as well, plus some other light rewording.)